apps/web/hooks/useChart.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useRef, useState } from "react";
```

Comment:
I think this hook should be marked as client as well. You use browser APIs here and state management